docs/source/usersguide/geometry.rst

```diff
@@ -183,9 +183,13 @@ the :class:`openmc.Cell` class::
   fuel.fill = uo2
   fuel.region = pellet
 
+  # This cell will be filled with void on export to XML
+  gap = openmc.Cell(region=pellet_gap)
+
 In this example, an instance of :class:`openmc.Material` is assigned to the
 :attr:`Cell.fill` attribute. One can also fill a cell with a :ref:`universe
-<usersguide_universes>` or :ref:`lattice <usersguide_lattices>`.
+<usersguide_universes>` or :ref:`lattice <usersguide_lattices>`. If you provide
+no fill to a cell, it will be filled with void on export to XML.
```

Comment:
Small wording suggestion for this last sentence:

"If no fill is provided to a cell, it will be filled with void by default."

Reply:
I have changed the wording as suggested.